radio/src/gui/480x272/lcd.h

```diff
@@ -133,7 +133,7 @@ void lcdDrawNumber(coord_t x, coord_t y, int32_t val, LcdFlags flags=0, uint8_t
 
 void putsModelName(coord_t x, coord_t y, char *name, uint8_t id, LcdFlags att);
 void putsStickName(coord_t x, coord_t y, uint8_t idx, LcdFlags att=0);
-void drawSwitch(coord_t x, coord_t y, swsrc_t swtch, LcdFlags flags=0);
+void drawSwitch(coord_t x, coord_t y, int32_t swtch, LcdFlags flags=0);
```

Comment:
Why isn't `lcdDrawBitmapPattern()` inlined into `layout1x1.cpp` like it is supposed to be, then the function `lcd->drawBitmapPattern()` would be linked!?